tests/phpunit/tests/script-modules/wpScriptModules.php

```diff
@@ -126,6 +126,33 @@ public function test_wp_dequeue_script_module() {
 		$this->assertTrue( isset( $enqueued_script_modules['bar'] ) );
 	}
 
+
+	/**
+	* Tests that a script module can be deregistered
+	* after being enqueued, and that will be removed
+	* from the enqueue list too.
+	*
+	* @ticket 60463
+	*
+	* @covers ::register()
+	* @covers ::enqueue()
+	* @covers ::deregister()
+	* @covers ::print_enqueued_script_modules()
+	*/
```

Comment:
Nit: I see that the PHPDoc isn't aligned according to WP coding styles:
```suggestion
	/**
	 * Tests that a script module can be deregistered
 	 * after being enqueued, and that will be removed
	 * from the enqueue list too.
	 *
	 * @ticket 60463
	 *
	 * @covers ::register()
	 * @covers ::enqueue()
	 * @covers ::deregister()
	 * @covers ::get_enqueued_script_modules()
	 */
```

See:

![Uploading Screenshot 2024-02-12 at 11.28.47.png…]()

Reply:
Done!